docs/user-guide/customization.rst

```diff
@@ -15,11 +15,38 @@ in a platform specific directory, which you can see the path for by running::
   >>> import hermes_core
   >>> hermes_core.print_config()  # doctest: +SKIP
 
+Using your own :file:`configrc` file
+=====================================
+To maintain your own customizations, you must place your customized :file:`configrc` inside the appropriate configuration folder (Which is based off the operating system you are working from). The `AppDirs module <https://github.com/sunpy/sunpy/blob/main/sunpy/extern/appdirs.py>`_  provided by the `sunpy` package is used to figure out where to look for your configuration file. 
 
-To maintain your own customizations place a copy of the default file into the *first* path printed above.
-Do not edit the default file directly as every time you install or update, this file will be overwritten.
+.. warning::
+    Do not just edit the default file directly as every time you install or update, this file will be overwritten.
 
-See below for the example config file.
+For example, on a Linux system, the configuration folder is :file:`~/.config/hermes_core/`. You can then place your customized :file:`configrc` file inside this folder.
+
+More specifically if you work in our devcontainer environment you can place your configuration file in this directory:
+
+.. code-block:: bash
+
+  /home/vscode/.config/hermes_core/
+
+You can also run the following code-block to see where to place it on your specific machine as well:
+
+.. doctest::
+
+  >>> from hermes_core import util
+  >>> print(util.config._get_user_configdir())
+
+  /home/vscode/.config/hermes_core/
+
+
+.. note:: 
+  To get more information on where to place your configuration file depending on your operating system, you can refer to the `AppDirs module docstrings <https://github.com/sunpy/sunpy/blob/1459206e11dc0c7bfeeeec6aede701ca60a8630c/sunpy/extern/appdirs.py#L165>`_. 
+
+
+To learn more about how to set-up your development environment see :ref:`dev_env`.
+
+See below (:ref:`configrc-sample`) for an example configuration file.
 
 .. _customizing-with-dynamic-settings:
 
```

Comment:
Could make changes to the section below, 
- fix reference to `sunpy.config`
- Also change to "You can also dynamically change most of the default settings". Add "One setting that cannot be changed is the location of the log file which is set on import".

Reply:
Just pushed a commit to fix these changes ^